wallet/storage/src/schema.rs

```diff
@@ -41,5 +41,7 @@ storage::decl_schema! {
         pub DBAddresses: Map<AccountDerivationPathId, Address>,
         /// Store for block/transaction entries
         pub DBTxs: Map<AccountWalletTxId, WalletTx>,
+        /// Store for wallet created transactions
+        pub DBSTxs: Map<AccountWalletSTxId, SignedTransaction>,
```

Comment:
`DBTxs` and `DBSTxs` look very similar. Something that makes the distinction more explicit would be nice.

Reply:
renamed